homeassistant/components/google_assistant/trait.py

```diff
@@ -302,7 +306,7 @@ class CameraStreamTrait(_Trait):
     def supported(domain, features, device_class, _):
```

Comment:
```suggestion
     def supported(domain, features: CameraEntityFeature, device_class, _):
```
Would this be a good change to test as well?

Reply:
I would do these changes in a separate PR